### ISSUE TITLE
Problem with Nintendont

### ISSUE THREAD

**Issue by mirober** [FIX94/Nintendont#361]
Hello everyone.
I have a white Nintendo Wii ver 4.3E , and I have already softmoded it by following this guide :
https://www.youtube.com/watch?v=6Hk5OHYD3Ic
I can perfectly play my Wii backups on Usb Loader GX , unfortunately I have a problem with my GC Games.
I have already installed Nintendont on my wii and st the Loader GX to use Nintendont to boot the GC games, and I have put my GC game (Mario kart : DD!) in a 8Gb thumb drive (Fat 32 and It's 32kb cluster) by using Discex and it's placed like this :
USB:/games/Mario Kart: Double Dash!! [GM4P01]/game.iso
(later I tried this :  USB:/games/GM4P01/game.iso)
However, when I boot the game using the loader gx I get this message : 
CheckForMultiGameAndRegion() = failed
![03e8d0c8-d6a7-11e6-9116-934f65fc5495](https://cloud.githubusercontent.com/assets/25012313/21781554/d1fbe2ca-d6a7-11e6-863a-caea698bdd89.jpg)
and when I boot the game by using the Homebrew chanel I get this message:

WARNING : USB:/games/ was not found
![20170110_110421](https://cloud.githubusercontent.com/assets/25012313/21804446/f2e3f534-d726-11e6-8e5b-0bd624259bf6.jpg)
and then the Nintendo freezes.
Later I tried using Dios Mios , I can actually select the game but the screen turns black without seeing the Dios Mios logo (I thought it should be mentioned)
My infomations are :  Nintendont v4.432      Dios-Mios v2.10 
and this is my loader GX credit page 
![20170110_120600 1](https://cloud.githubusercontent.com/assets/25012313/21805761/c7829fba-d72d-11e6-987e-b39928a203fb.jpg)

I don't know what I did wrong , can anybody help me?

**Comment by ArgonUK** [comment]
Is your backup PAL or NTSC-U or NTSC-J region?

My own backup region code is GM4E01, which is for USA / NTSC-U region. According to the error code, it might be that you specified incorrect region in the folder name.

**Comment by mirober** [comment]
my back up is NTSC-U.
and my wii is 4.3E (Europe) ..do you think that's the issue here?

**Comment by FIX94** [comment]
both dios-mios and nintendont obviously cant access your usb fat partition so you might want to properly re-format it to fat32 as primary partition and mbr as partition table.

**Comment by ArgonUK** [comment]
You specified incorrect disc code in the game folder name. Change it to GM4E01, so the total path is as follows:
USB:/games/Mario Kart: Double Dash!! [GM4E01]/game.iso

**Comment by FIX94** [comment]
path names dont matter, if it cant even see the games folder when starting nintendont then you know its a problem with the usb formatting.

**Comment by mirober** [comment]
I have tried the new path , but nothing change .
I still have the same problem

**Comment by mirober** [comment]
do you have any formating software  that you can recommend for me?

**Comment by mirober** [comment]
I was wrong . my game is the Pal version.
I also tried the path you recommended but nothing changed.
the only thing left to try is reformating my USB with a software not
windows .
do you have any good formating software to recommend?

On Jan 14, 2017 17:28, "ArgonUK" <notifications@github.com> wrote:

Is your backup PAL or NTSC-U or NTSC-J region?

My own backup region code is GM4E01, which is for USA / NTSC-U region.
According to the error code, it might be that you specified incorrect
region in the folder name.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/FIX94/Nintendont/issues/361#issuecomment-272638783>,
or mute
the thread

**Comment by ArgonUK** [comment]
Any will do. I just googled it up last time.

**Comment by mirober** [comment]
tried your suggestion,unfortunately there's no result.

**Comment by carnage702** [comment]
im guessing your hdd just isnt compatible at all or your game.iso is called game.iso.iso and has extensions hidded on your pc?

**Comment by mirober** [comment]
I'm sure it's not game.iso.iso.
maybe my hdd is bot compatible,but I should mention that I can run my Wii
backups  perfectly fine from the same hdd.

**Comment by briantho** [comment]
Hi, what's the brand and model of your flash drive? I used to have the same error message, the checkformultigameandregion () failed. using usb loader gx, on my Seagate laptop drive that I put inside a usb enclosure. The remedy to this solution was for me to get a new drive enclosure since there was some weird issue using my old enclosure. However since your using a flash drive, my solution won't work. I'd recommend doing some testing with another drive/usb device and use easeus partition manager to format to fat32. Also, have you tried older versions of nintendont. May help, as when I first experienced this issue, I had done an upgrade to 4.432 from an old version (sorry I forgot which one). The old version worked with my original usb drive, and 4.432 would not. Anyway, just let me know if anything helps.

**Comment by mirober** [comment]
My flash drive is Kingston 8Gb ,I'll try formating it with the software you
recommended. but where can I find an older version of Nintendont.

**Comment by nativesprague** [comment]
Would like to chime in and just say that I've also been suffering from this error on Nintendont 4.432 - I'm using a PNY 32GB Attache USB 2.0 Flash Drive to store and play GC games. I usually just ignored the error because when I would load the game a second time, it'd always play just fine... but when I received the error earlier today, it totally wiped everything from my flash drive. 

Gonna try out older Nintendont versions and report back.

**Comment by mirober** [comment]
thanks for the tip, where can I find older versions of Nintendont?

**Comment by nativesprague** [comment]
Here ya go mate, versions [2.229 & 2.277](https://mega.nz/#!apNmVYKK!12TDzZvAIRkVqo_id2veDviwhzBMykJTqXBH9jGfNR4).

**Comment by carnage702** [comment]
flashdrives are never recomended for homebrew in general.

**Comment by briantho** [comment]
Hi, everybody, I just tried loading nintendont 4.434 on my Kingston 16gb DT101 g2 flash drive, and everything works as it should, also tried from usb loader gx, and couldn't reproduce the problem you guys are having. as some suggestions I'd try on another usb drive and still see if you have issues, particularly a usb hard drive, (If not then another flash drive). Also, how do you have nintendont settings set up. I have mine at default, with only "memory card multi" turned on. I tested with clean dumps of Mario kart double dash, and melee, and Mario party 6. Also, what is the exact model of flash drive you are using, I've got a few kingstons lying around, maybe one of them is the same model as yours.

**Comment by mirober** [comment]
but I can run my wii backups just fine from the same usb!

**Comment by carnage702** [comment]
nintendont is  totaly diferent codewise to how usb works compared to  other loaders, becuase  some restraints.

**Comment by nativesprague** [comment]
Alright, would just like to post an update on the situation for anyone else that may be having this issue as well.

Through what I've learned from troubleshooting this problem the last few days, I can almost say with certainty that the "CheckForMultiGameAndRegion()" error is simply caused by bad compatibility with the USB flash drive being used (in my case, a 32GB PNY Attache).

I purchased a cheap SATA to USB adapter and tested it with an old 120GB Seagate Barracuda I had laying around, and ALL these issues vanished. (And the upgrade from 32GB to 120GB was nice...)
